addons/web_editor/static/src/js/editor/image_processing.js

```diff
@@ -0,0 +1,132 @@
+odoo.define('web_editor.image_processing', function (require) {
+'use strict';
+
+// Fields returned by cropperjs 'getData' method, also need to be passed when
+// initializing the cropper to reuse the previous crop.
+const cropperDataFields = ['x', 'y', 'width', 'height', 'rotate', 'scaleX', 'scaleY'];
+const modifierFields = [
+    'filter',
+    'quality',
+    'mimetype',
+    'originalId',
+    'originalSrc',
+    'resizeWidth',
+    'aspectRatio',
+];
+/**
+ * Applies data-attributes modifications to an img tag and returns a dataURL
+ * containing the result. This function does not modify the original image.
+ *
+ * @param {HTMLImageElement} img the image to which modifications are applied
+ */
+async function applyModifications(img) {
+    const data = Object.assign({
+        filter: '#0000',
+        quality: '95',
```

Comment:
If the second parameter of `toDataURL` is omitted and the mimetype supports optimization, the quality value will be .92

I chose 95 because we used to cap the quality at 95 in the image optimization dialog. I don't mind changing it if you think it's better to stick to the browser's default value.

Reply:
No, 95 ok. I just wonder why we chose 95 in the first place and why the browser spec chose 92.